meetings/2019-10-29.md

```diff
@@ -0,0 +1,95 @@
+# Code Curious Elixir Meeting 2019-10-29
+
+## Attendees
+
+- Jens (moderating)
+- Nicolo
+- Markus
+- Max
+- Tobi
+- Daniel
+
+## Topics
+
+- recap last TODOs
+- decide on workshop and audience
+- find date for workshop
+
+## Organizational matters
+
+- low attendance: check who's interested in coaching and organizing
+  - people are sick
+  - we are enough people to organize the workshop
+  - once we know about the workshop: make call for coaches
+- early Doodle and clear announcement helps for attendance
+- the work on the guide is done via PRs (merge whenever it makes sense)
+
+## Decide on workshop
+
+### Which audience
+
+- start with intermediate:
+  - works best for first workshop
+  - quick vote: we start with intermediate
+- "intermediate" means confident with with computers
+  - this of course includes juniors
+
+### Which content
+
+- recap:
+  - nerves project (we stopped looking at it)
```

Comment:
"stopped looking at it" seems wrong, we always talked about it but we did not have any further input and work done on it.

Reply:
👍 rephrased it.